docs/skopeo-sync.1.md

```diff
@@ -0,0 +1,80 @@
+% skopeo-sync(1)
+
+## NAME
+skopeo\-sync - Copy images from one or more repositories to a user specified destination.
+
+## SYNOPSIS
+**skopeo sync** [**--source-yaml**] _source_ _destination_
+
+## DESCRIPTION
+Copy images from one or more repositories to a user specified destination.
+
+Useful to keep in sync with a local container registry mirror. It can also be used to populate registries running inside of air-gapped environments.
+
+_source_ can be:
+ - a repository hosted on a container registry (eg: docker://registry.example.com/busybox)
+ - a local directory (eg: dir:/media/usb/).
+ - (when **--source-yaml** is specified) a YAML file with a list of source images from different container registries.
+
+When the source location is a container repository and no tags are specified,  **skopeo sync** will copy all the tags associated to the source image.
+
+_destination_ can be either a container registry (eg: docker://my-registry.local.lan) or a local directory (eg: dir:/media/usb).
+
+When _destination_ is a local directory, one directory per 'image:tag' will be created.
+
+## OPTIONS
+
+**--authfile** _path_
+
+Path of the authentication file. Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `podman login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
+
+**--remove-signatures** do not copy signatures, if any, from _source-image_. Necessary when copying a signed image to a destination which does not support signatures.
+
+**--sign-by=**_key-id_ add a signature using that key ID for an image name corresponding to _destination-image_
+
+**--source-yaml** Interpret _source_ as a YAML file with a list of images from different container registries
```

Comment:
Please mention that it only works with the `docker://` transport. Might need to be updated in the help text as well.

Reply:
Still valid.